hudi-client/hudi-spark-client/src/main/scala/org/apache/spark/sql/hudi/SparkAdapter.scala

```diff
@@ -92,4 +95,31 @@ trait SparkAdapter extends Serializable {
    * ParserInterface#parseMultipartIdentifier is supported since spark3, for spark2 this should not be called.
    */
   def parseMultipartIdentifier(parser: ParserInterface, sqlText: String): Seq[String]
+
+  def isHoodieTable(table: LogicalPlan, spark: SparkSession): Boolean = {
```

Comment:
Is there any difference with **hoodieSqlCommonUtils.isHoodieTable**? I see sometimes we use **adapter.isHoodieTable**, sometimes use **hoodieSqlCommonUtils.isHoodieTable**

Reply:
> Is there any difference with **hoodieSqlCommonUtils.isHoodieTable**? I see sometimes we use **adapter.isHoodieTable**, sometimes use **hoodieSqlCommonUtils.isHoodieTable**

in fact **hoodieSqlCommonUtils.isHoodieTable** method is used in v1 codebase to judge if a table is a hoodie table in v1 codebase , but **adapter.isHoodieTable** method is to judge if a table is a hoodie table in v2 codebase, change the name would be better to understand.

Reply:
Thanks for your reply, and please correct me if I'm wrong. Can we just move the method implementation out from SparkAdapter to Spark2Adapter, Spark2Adapter can judge if a table is a hoodie table in v1 codebase, while Spark3Adapter can judge it by v2 codebase, by this, I think the method **hoodieSqlCommonUtils.isHoodieTable** can be simply removed?

Looking forward this PR can be merged as soon as possible, this is an excellent work as we can support many other features with DSV2 based on this. :)